src/JitInterface/src/JitConfigProvider.cs

```diff
@@ -78,6 +81,11 @@ public string GetStringConfigValue(string name)
             return String.Empty;
         }
 
+        public bool IsModuleInstrumented(ModuleDesc module)
```

Comment:
This API needs to go to Compilation or the CompilationModuleGroup. JitConfigProvider only wraps JIT flags and COMPlus options that are valid for all compiled methods. It doesn't deal with policies.

Reply:
I've put it on Compilation to avoid variance between compile a file and compile a single method codegen strategies.